fiddler/manifest.json

```
@@ -0,0 +1,46 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ee617671-508e-4bb3-ba25-8815b11a16aa",
+  "app_id": "fiddler",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+    "Category::Alerting",
+    "Category::Monitoring",
+    "Category::Metrics"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "",
+    "title": "Fiddler",
+    "media": []
+  },
+  "author": {
+    "support_email": "rajeev@fiddler.ai"
```

The manifest.json v2 documentation indicates a few more fields are required: https://docs.datadoghq.com/developers/integrations/check_references/?tab=manifestversion2#manifest-file

```suggestion
    "support_email": "rajeev@fiddler.ai",
    "homepage": "<homepage>",
    "name": "<author_name>",
    "sales_email": "<sales_email>"
```

---

Done